modules/distr-tracing-otel-config-collector.adoc

```diff
@@ -365,3 +365,56 @@ The Logging exporter prints data to the standard output.
           exporters: [logging]
 ----
 <1> Verbosity of the logging export: `detailed` or `normal` or `basic`. When set to `detailed`, pipeline data is verbosely logged. Defaults to `normal`.
+
+[id="extensions_{context}"]
+=== Extensions
+
+[id="basicauth-extension_{context}"]
+==== BASICAUTH extension
+
+The BasicAuth extension can be used as an authenticator on receivers and exporters.
+Client authentication and server authentication for the BasicAuth extension are configured in separate sections in the OpenTelemetry Collector custom resource.
+
+* Support level: link:https://access.redhat.com/support/offerings/techpreview[Technology Preview]
+* Supported signals: traces, metrics, logs
+
+.OpenTelemetry Collector custom resource with client and server authentication configured for the BasicAuth extension
+[source,yaml]
+----
+  config: |
+    extensions:
+      basicauth/server:
+        htpasswd:
+          file: .htpasswd <1>
+        inline: |
+          ${env:BASIC_AUTH_USERNAME}:${env:BASIC_AUTH_PASSWORD} <2>
+
+      basicauth/client:
+        client_auth:
+          username: username <3>
+          password: password <4>
+
+    receivers:
+      otlp:
+        protocols:
+          http:
+            auth:
+              authenticator: basicauth/server <5>
+    exporters:
+      otlp:
+        auth:
+          authenticator: basicauth/client <6>
+
+service:
+  extensions: [basicauth/server, basicauth/client]
+  pipelines:
+    traces:
+      receivers: [otlp]
+      exporters: [otlp]
+----
+<1> Here the BasicAuth extension is configured as a server authenticator that reads credentials from an `.htpasswd` file.
+<2> Here the BasicAuth extension is configured as a server authenticator to read the credentials from an inline string that consists of environment variables.
+<3> The client username is configured as a client authenticator for the BasicAuth extension.
+<4> The client password is configured as a client authenticator for the BasicAuth extension.
+<5> Here the authenticator configuration is assigned to an OTLP receiver.
+<6> Here the authenticator configuration is assigned to an OTLP exporter.
```

Comment:
I've had a second look at my own applied suggestion:
```suggestion
<1> The BasicAuth extension can be configured as a server authenticator that reads credentials from a `.htpasswd` file.
<2> The BasicAuth extension can be configured as a server authenticator to read the credentials from an inline string that consists of environment variables.
<3> The client username is configured as a client authenticator for the BasicAuth extension.
<4> The client password is configured as a client authenticator for the BasicAuth extension.
<5> The authenticator configuration can be assigned to an OTLP receiver.
<6> The authenticator configuration can be assigned to an OTLP exporter.
```

Reply:
I changed <2> to client.